core/src/main/resources/error/error-classes.json

```diff
@@ -1973,6 +1973,12 @@
       "The sum of the LIMIT clause and the OFFSET clause must not be greater than the maximum 32-bit integer value (2,147,483,647) but found limit = <limit>, offset = <offset>."
     ]
   },
+  "TABLE_NOT_SUPPORTED_STATISTIC" : {
```

Comment:
Could you add a sub-class of `UNSUPPORTED_FEATURE`, please.

Reply:
Thanks for remind, done.